apps/console/src/public/deployment.config.json

```diff
@@ -309,7 +309,7 @@
         },
         "isGroupAndRoleSeparationEnabled": true,
         "isClientSecretHashEnabled": false,
-        "productName": "Identity Server",
+        "productName": "WSO2 Identity Server",
```

Comment:
Let's do the same change for myaccount config.

Reply:
Missing configurations in `myaccount` application is added in commit 6f5d9f8. Thanks!